addons/sale_project/tests/test_sale_project.py

```diff
@@ -93,6 +93,12 @@ def setUpClass(cls):
         # Create partner
         cls.partner = cls.env['res.partner'].create({'name': "Mur en béton"})
 
+    def test_task_create_sol_ui(self):
+        self.start_tour('/web', 'task_create_sol_tour', login='admin')
+
+    def test_project_create_sol_ui(self):
+        self.start_tour('/web', 'project_create_sol_tour', login='admin')
```

Comment:
Cannot we already create the project and task to avoid doing it inside the tour? 🧐 

By doing that we could gain many times and reduce the number of steps and probably merge the both tours together.

What do you think?